client/trino-jdbc/src/test/java/io/trino/jdbc/TestTrinoDatabaseMetaData.java

```diff
@@ -68,13 +67,15 @@
 
 import static com.google.common.base.Verify.verify;
 import static com.google.common.collect.ImmutableList.toImmutableList;
+import static com.google.common.collect.ImmutableMultiset.toImmutableMultiset;
 import static com.google.common.collect.ImmutableSet.toImmutableSet;
 import static com.google.common.collect.Iterables.getOnlyElement;
 import static io.airlift.testing.Assertions.assertContains;
 import static io.trino.jdbc.TestingJdbcUtils.array;
 import static io.trino.jdbc.TestingJdbcUtils.assertResultSet;
 import static io.trino.jdbc.TestingJdbcUtils.list;
 import static io.trino.jdbc.TestingJdbcUtils.readRows;
+import static io.trino.plugin.hive.util.MultisetAssertions.assertMultisetsEqual;
```

Comment:
How about moving this method to `trino-testing` or somewhere? Feel free to ignore or handle in follow-up.

Reply:
good idea!